server/src/main/java/org/apache/druid/indexing/overlord/supervisor/Supervisor.java

```diff
@@ -93,4 +94,10 @@ default Boolean isHealthy()
   LagStats computeLagStats();
 
   int getActiveTaskGroupsCount();
+
+  /** Handoff the task group with id=taskGroupId the next time the supervisor runs regardless of task run time*/
```

Comment:
Some rephrase + Druid javadoc styling
```suggestion
  /**
   * Marks the given task groups as ready for segment hand-off irrespective of the task run times.
   * In the subsequent run, the supervisor initiates segment publish and hand-off for these task groups and rolls over their tasks.
   * taskGroupIds that are not valid or not actively reading are simply ignored.
   */
```